com.github.tno.synthml.setup

```diff
@@ -240,8 +240,8 @@
   </setupTask>
   <setupTask
       xsi:type="setup:EclipseIniTask"
-      option="-Doomph.redirection.com.github.tno.pokayoke.git"
-      value="=file:/X:/PokaYoke/Oomph/com.github.tno.pokayoke.setup->${com.github.tno.pokayoke.git.clone.location|uri}/com.github.tno.pokayoke.setup"
+      option="-Doomph.redirection.com.github.tno.synthml.git"
+      value="=https://raw.githubusercontent.com/TNO/SynthML/${scope.project.stream.name}/com.github.tno.synthml.setup->${com.github.tno.synthml.git.clone.location|uri}/com.github.tno.synthml.setup"

```

Comment:
If I fill in `main` for `${scope.project.stream.name}`, I get `https://raw.githubusercontent.com/TNO/SynthML/main/com.github.tno.synthml.setup`, which gives a 404. `https://raw.githubusercontent.com/TNO/SynthML/refs/heads/main/com.github.tno.pokayoke.setup` does work.